include/crocoddyl/multibody/contacts/contact-1d.hxx

```diff
@@ -121,4 +145,24 @@ void ContactModel1DTpl<Scalar>::set_reference(const Scalar reference) {
   xref_ = reference;
 }
 
+template <typename Scalar>
+void ContactModel1DTpl<Scalar>::set_type(const pinocchio::ReferenceFrame type) {
+  type_ = type;
+}
+
+template <typename Scalar>
+const pinocchio::ReferenceFrame ContactModel1DTpl<Scalar>::get_type() const {
+  return type_;
+}
+
+template <typename Scalar>
+void ContactModel1DTpl<Scalar>::set_mask(const Contact1DMaskType mask) {
+  mask_ = mask;
```

Comment:
If you change to `LOCAL_WORLD_ALIGNED`, then define the identity matrix as pointed out above.